src/stem-tracker.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+var Stem = require("./stem");
+
+/**
+ * Stem Tracking Utility.
+ *
+ * Handles the stems for a given editor.
```

Comment:
This line doesn't add much info - instead of it, describe what a "stem" is, and what it means to track them